payment-manager/src/main/java/com/qpay/paymentmanager/model/usertype/UserType.java

```diff
@@ -0,0 +1,6 @@
+package com.qpay.paymentmanager.model.usertype;
+
+public enum UserType {
```

Comment:
this Enum is common for lots of microservices. What you need to do is to create a new project `libs` and move common models there. For now only this class. Then import this project in your local `build gradle` as a dependency.